newsheadlines.py

```diff
@@ -0,0 +1,63 @@
+import click
+import requests
+#not in range expection error
+class NotInRangeError(Exception):
+    pass
+
+def news_source_menu():
+    click.echo(click.style('Select your favorate source of News', fg='blue'))
+    click.echo(click.style('1. FOX-News', fg='blue'))
+    click.echo(click.style('2. CNN', fg='blue'))
+    click.echo(click.style('3. Al- Jazeera - english', fg='blue'))
+    click.echo(click.style('4. BBC-Sport', fg='blue'))
+    click.echo(click.style('-' * 100, fg='blue'))
+
+
+def news_board():
+    click.echo('\n')
+    click.echo(click.style('*' * 100, fg='blue'))
+    click.echo(click.style('*' + ' ' * 98 + '*', fg='blue'))
+    click.echo(click.style('*' + ' ' * 98 + '*', fg='blue'))
+    click.echo(click.style('*' + ' ' * 32 + 'News HeadLines ' + ' ' * 32 + '*', fg='blue'))
+    click.echo(click.style('*' + ' ' * 22 + 'The latest and Live breaking news headlines ' + ' ' * 12 + '*', fg='blue'))
+    click.echo(click.style('*' + ' ' * 98 + '*', fg='blue'))
+    click.echo(click.style('*' + ' ' * 98 + '*', fg='blue'))
+    click.echo(click.style('*' * 100, fg='blue'))
+    click.echo('\n')
+
+
+@click.command()
+def cli():
+
+    news_board()
+    news_source_menu()
+    news_source_number = input('> Enter the number of your Favorate Source of News here: ')
+
+    if news_source_number not in ("1", "2", "3", "4"):
+        raise NotInRangeError("Please select numbers between 0 and 4")
+   
```

Comment:
There is a lot of duplications of the URL here.
Consider putting the duplicated parts in variables